ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -366,6 +370,16 @@ def verify_operator_succeeded(
             )
         self.take_screenshot()
 
+    def create_lso_catalog_source(self):
+        """
+        Setup the necessary resources for enabling local storage.
+        [for non Ga'd OCP version]
```

Comment:
What is missing? Can you please elaborate more a bit.
We are already handling non_ocp_ga version in setup_local_storage() here https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/helpers/lso_helpers.py#L46?

Reply:
In order to enable Local storage operator on OCP [no GA'd], we need to use the following command.
oc create -f https://raw.githubusercontent.com/red-hat-storage/ocs-ci/master/ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml

Reply:
Technically the code to setup catalog source for unreleased LSO is there, but it's not possible to call it independently. Would it make sense to move this code out into separate function?

Reply:
We already handled the creation of the local-storage-optional operators yaml for non_ocp_ga version in setup_local_storage() https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/helpers/lso_helpers.py#L46
I see you are creating a new wrapper create_lso_catalog_source and calling the same setup_local_storage again. why dont we directly call  setup_local_storage?
Not sure if I understand this change correctly.